src/tv2_afvd_showstyle/migrations/index.ts

```diff
@@ -265,6 +267,12 @@ export const showStyleMigrations: MigrationStepShowStyle[] = [
 	renameTableColumn('1.7.9', 'OverlayShowMapping', 'GraphicsSetup', 'GfxSetup'),
 	renameTableId('1.7.9', 'OverlayShowMapping', 'GfxShowMapping'),
 
+	/**
+	 * 1.8.0 Move SelectedGfxSetupName to GFX Defaults
+	 */
+	moveSelectedGfxSetupNameToGfxDefaults('1.8.2', 'SelectedGfxSetupName', 'GfxDefaults'),
+	moveSelectedGfxSetupNameToGfxDefaultsInVariants('1.8.2', 'SelectedGfxSetupName', 'GfxDefaults'),
```

Comment:
It doesn't really make sense that you have to specify parameters such as `SelectedGfxSetupName` and `GfxDefaults` when the method name explicit states that that's what is using.

Reply:
Fixed in the newest commit!